packages/bolt-interactions/package.json

```diff
@@ -9,14 +9,19 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "@slack/bolt": "^1.4.1",
     "@slack/types": "^1.3.0",
     "@slack/web-api": "^5.6.0",
     "@types/uuid": "^3.4.6",
     "uuid": "^3.3.3"
   },
+  "peerDependencies": {
+    "@slack/bolt": ">=1.0.0 <3.0.0"
```

Comment:
This caused weirdness actually trying to use it especially with mismatched types between 1 and 2.